src/sqlparser/src/parser.rs

```diff
@@ -1324,6 +1324,23 @@ impl Parser {
         }
     }
 
+    /// Call `f` with the current token and returns the result if the current token is not one of
+    /// the expected keywords, or consume it if it is
+    pub fn expect_one_of_keywords_or<F>(
```

Comment:
Why not simply `expect_one_of_keywords`?

Reply:
To customize the error message

Reply:
I feel it not very necessary and has inconsistent style. Maybe just simply don't customize error message, or 
```
expect_one_of_keywords(..).map_err(|e| match e {
    ParserError::TokenizerError(_) => unreachable!(),
    ParserError::ParserError(e) => {
        ParserError::ParserError(format!("Unrecognized EXPLAIN option: {e}"))
    }
})?;
```